com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -24,11 +27,16 @@ public class NetworkConfig
         public NetworkTransport NetworkTransport = null;
 
         /// <summary>
-        /// A list of SceneNames that can be used during networked games.
+        /// The list of SceneNames built from the RegisteredSceneAssets list
         /// </summary>
-        [Tooltip("The Scenes that can be switched to by the server")]
+        [HideInInspector]
         public List<string> RegisteredScenes = new List<string>();
 
+#if UNITY_EDITOR
```

Comment:
why there's this `#if UNITY_EDITOR` difference between `List<string>` and `List<SceneAsset>`?

Reply:
Its because SceneAssets are an editor only thing

Reply:
This is because [UnityEditor.SceneAsset](https://docs.unity3d.com/ScriptReference/SceneAsset.html) is an editor only asset.  This will all be replaced in [PR-967](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/967).
However, in the event we don't make it to PR-967 this change is here to change the scene reference by name as a string to scene reference by SceneAsset (which during NetworkManager.OnValidate extracts the SceneAsset.name and places it in the `public List<string> RegisteredScenes = new List<string>();`